modules/sitestudio_config_management/composer.json

```diff
@@ -5,7 +5,7 @@
     "license": "GPL-2.0-or-later",
     "require": {
         "php": ">=7.4",
-        "acquia/cohesion": "7.4.x-dev",
+        "acquia/cohesion": "~7.4",
```

Comment:
Here as well, this need to be updated as `~7.4.0`.

Reply:
Done